### DISCUSSION TITLE
What is assumed as "the specification" in YAML file?

### DISCUSSION THREAD

**Discussion by Masa8106** [camaraproject/KnowYourCustomer#154] [Q&A] [answered]
Hi

There is a following text in KYC Match YAML file.
_Unless otherwise captured in the specification, score will use the Jaro-Winkler distance algorithm for all countries._

My question is what is assuemed as "the specification" here.
Is it KYC Match YAML file itself? and/or 
Is it any documents which API provider provides developers to explain their API service?

Thank you.

Masa
KDDI

**Comment by GillesInnov35** [comment, marked as answer]
hello, as the yaml file must be the Camara release version, this specification could not be included in the yaml. I think it could be part of the Geting started as it's published by the API provider on the exposition API Gateway.
BR
Gilles

**Reply by Masa8106** [reply]
Thanks for sharing your view. Please let me collect other views by the next meeting.

**Comment by rartych** [comment]
I agree the word "specification" may be confusing here.
My proposal is to change it to "implementation notes", as change of the algorithm is done by operator in specific implementation.

**Reply by Masa8106** [reply]
Thanks for suggestion. Let me raise a issue and PR for the rewording.
